### DISCUSSION TITLE
Fooocus 2.3.0 images have too much lights and look like "plastic"

### DISCUSSION THREAD

**Discussion by gorrosh** [huggingface/diffusers#7439] [Q&A]
Hi,
First thanks for the amazing work around the new update.
I'm having trouble getting similar images as before, with more quality, using the new interface.
i tried all the different presets, with combination of base models i had before and it's impossible to get thje same results as the previous verions
i kept the prompt and all the settings for my previous images so i can set the exact same parameters but the image looks too bright (too much light on character) and skin looks more like plastic. general vision is lower quality.
is there any way to get this working as good as before ?

**Comment by tolgacangoz** [comment]
It seems that you are in the wrong repo.
